Makefile

```diff
@@ -26,7 +26,7 @@ else
 	$(eval GIT_BRANCH_CLEAN := unknown)
 endif
 
-override LIBS += $(shell $(PKG_CONFIG) --libs glib-2.0)
+override LIBS += $(shell $(PKG_CONFIG) --libs glib-2.0) -ldl
```

Comment:
I think you'd also need to add the pkg-config call for the seccomp lib here.

Similar is true for the CFLAGS below. Somewhere you'd need to add the equivalent of `pkg-config --cflags libseccomp`